### ISSUE TITLE
Regarding the issue of using rerank in the code

### ISSUE THREAD

**Issue by xl-Zzzzzzz** [1KK077/IDKL#4]
First of all, thank you for your contribution to VI-ReID. We have carefully read your article and reproduced the work using the code you provided. We have found that you used rerank in your code, specifically in your testing on the SYSU-MM01 and LLCM datasets, but you did not mention it in the preprint. As is well known, work in the VI ReID field generally does not use the results after rerank as a comparative indicator. We hope you can explain this.

**Comment by alehdaghi** [comment]
Did you get the same results as the paper?

**Comment by YoungKyle** [comment]
We set the rerank parameter to false, the rank-1 is only about 71% on SYSU dataset.

**Comment by 1KK077** [comment]
Hello, thank you very much for your concern. This code mainly provides the main module functions of the paper for reference, and is not a complete code. Some tricks are omitted and optimizer hyperparameters are modified, etc. The actual code does not use rerank and is based on another SOTA model to build the baseline. The complete code will be released soon. Thank you for your patience.

**Comment by echo9958** [comment]
> Hello, thank you very much for your concern. This code mainly provides the main module functions of the paper for reference, and is not a complete code. Some tricks are omitted and optimizer hyperparameters are modified, etc. The actual code does not use rerank and is based on another SOTA model to build the baseline. The complete code will be released soon. Thank you for your patience.

If the baseline was constructed based on another SOTA model, are the results of the ablation experiments in Table 4 in the preprint using two different baselines? Or is the accuracy of the SOTA model 66.47% mAP? This is very confusing to me and I hope you can explain.
<img width="941" alt="pic" src="https://github.com/1KK077/IDKL/assets/45514395/f8ec5977-997f-4dd7-821d-0a4d46fc46af">

**Comment by 1KK077** [comment]
> > Hello, thank you very much for your concern. This code mainly provides the main module functions of the paper for reference, and is not a complete code. Some tricks are omitted and optimizer hyperparameters are modified, etc. The actual code does not use rerank and is based on another SOTA model to build the baseline. The complete code will be released soon. Thank you for your patience.
> 
> If the baseline was constructed based on another SOTA model, are the results of the ablation experiments in Table 4 in the preprint using two different baselines? Or is the accuracy of the SOTA model 66.47% mAP? This is very confusing to me and I hope you can explain. <img alt="pic" width="941" src="https://private-user-images.githubusercontent.com/45514395/339702713-f8ec5977-997f-4dd7-821d-0a4d46fc46af.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTgzODE4MjQsIm5iZiI6MTcxODM4MTUyNCwicGF0aCI6Ii80NTUxNDM5NS8zMzk3MDI3MTMtZjhlYzU5NzctOTk3Zi00ZGQ3LTgyMWQtMGE0ZDQ2ZmM0NmFmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjE0VDE2MTIwNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdhMTU3M2ZjNGMzNjZlODJlNDBkZGJiYzQ5NmY0YTMzNmZlMTMxNGYxOWVmNmRkYmQ5YjY4ZmRiZTIzNjAyN2UmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.dxm0NX3NcPtoXMI2CkD9LTB1YwJcGpD15H3qvLmjih8">

Hello, thank you very much for your concern. The 66.47% mAP reported in Table 4 represents the baseline result of ResNet-50. However, we enhanced this baseline using several SOTA tricks. These include the channel exchange trick from Mang Ye et al., the ResNet block attention enhancement trick from Yukang Zhang et al., our own trick of Adam regularization parameter fitting boundary training, and local image enhancement methods, among others. And the Adam learning rate is 3.5 * 10^-5, without special modifications. We hope this clarifies your confusion.

**Comment by GG0614** [comment]
> > 你得到的结果和论文一样吗？
> 
> Our reproducible results show that in the SYSU dataset, the performance with rerank is approximately 81%, while without rerank, it is only about 71%.

Hello,I am trying to reproduce this result too, but I meet with several problems in the dataset,Do you add other file into his code?Or can you share your project structure?

**Comment by 4fouroy4** [comment]
Dear Author,

Thank you for sharing your codes. I reproduced the following metrics when setting Rerank = True:


SYSU Evaluation - mAP: 80.0469, Rank-1: 81.8380, Rank-5: 94.5070

When setting Rerank = False, I obtained:

SYSU Evaluation - mAP: 68.8699, Rank-1: 70.5601, Rank-5: 90.4549

I noticed from a previous discussion that comparing results with and without reranking might not be entirely fair. Could you please clarify why you chose to compare these two settings or provide additional context about how you view the results?

Thank you, and I look forward to your explanation.

**Comment by wzhao97** [comment]
My reproduction results are as follows：
 all num-shot:1 r1 precision = 70.42 , r10 precision = 95.48 , r20 precision = 98.33, mAP = 68.41
 all num-shot:10 r1 precision = 74.74 , r10 precision = 96.98 , r20 precision = 98.69, mAP = 63.56
 indoor num-shot:1 r1 precision = 79.92 , r10 precision = 97.56 , r20 precision = 99.19, mAP = 82.61
 indoor num-shot:10 r1 precision = 86.04 , r10 precision = 99.28 , r20 precision = 99.80, mAP = 78.73





> Dear Author,
> 
> Thank you for sharing your codes. I reproduced the following metrics when setting Rerank = True:
> 
> SYSU Evaluation - mAP: 80.0469, Rank-1: 81.8380, Rank-5: 94.5070
> 
> When setting Rerank = False, I obtained:
> 
> SYSU Evaluation - mAP: 68.8699, Rank-1: 70.5601, Rank-5: 90.4549
> 
> I noticed from a previous discussion that comparing results with and without reranking might not be entirely fair. Could you please clarify why you chose to compare these two settings or provide additional context about how you view the results?
> 
> Thank you, and I look forward to your explanation.
